README.md

```diff
@@ -338,12 +371,22 @@ infos       | Display the name of a node and show the associated scripts
 go          | Jump to a node
 var         | Display or alter the value of a variable from the game state
 
+### 8.4. Debugging The Longest Journey ###
+
+The debug console can be triggered using Ctrl + d. Use the `help` command to
+display a list of the available commands.
+
+The most useful commands are:
+
+Command     | Description
+----------- | ------------------------------------------------------------
+*TODO*      | *TODO*
```

Comment:
Remove this part if you don't want to document the commands. There is already some debugging documentation on the wiki anyways. No need to duplicate it here.

Reply:
I have copied it from the other games, assuming it's required – I can remove it, but I guess I can do some research as well to fill it up – are there any pointers to look at if I want to document it (assuming there are TLJ/Stark-exclusive debug commands there ofc :)).

Reply:
This wiki page has a list of useful debug commands: http://wiki.residualvm.org/index.php/TLJ_Debugging

The help command will also list all the commands that work with TLJ.